recipes/newrelic/infrastructure/ohi/elasticsearch/debian.yml

```diff
@@ -122,45 +122,45 @@ install:
         - |
           if [ {{.NR_CLI_API_USE_SSL}} == false ]; then
             sudo tee -a /etc/newrelic-infra/integrations.d/elasticsearch-config.yml > /dev/null <<"EOT"
-          integration_name: com.newrelic.elasticsearch
-
-          instances:
-            - name: elasticsearch
-              command: all
-              arguments:
-                cluster_environment: staging
-                config_path: {{.NR_CLI_API_CONFIG_PATH}}
-                hostname: {{.NR_CLI_API_HOSTNAME}}
-                username: {{.NR_CLI_API_USERNAME}}
-                password: {{.NR_CLI_API_PASSWORD}}
-                port: {{.NR_CLI_API_PORT}}
-                timeout: 30
-                use_ssl: false
-                indices_regex:
-                collect_indices: true
-                collect_primaries: true
+          integrations:
+            - name: nri-elasticsearch
+              env:
+                CLUSTER_ENVIRONMENT: staging
+                CONFIG_PATH: {{.NR_CLI_API_CONFIG_PATH}}
+                HOSTNAME: {{.NR_CLI_API_HOSTNAME}}
+                USERNAME: {{.NR_CLI_API_USERNAME}}
+                PASSWORD: {{.NR_CLI_API_PASSWORD}}
+                PORT: {{.NR_CLI_API_PORT}}
+                TIMEOUT: 30
+                USE_SSL: false
+                INDICES_REGEX:
```

Comment:
this is weird but it was already empty

Reply:
Being the user I would prefer a comment to explain why that is empty and how I can use such parameter. The same comment applies to other env in different recipes

Reply:
yes, that's true.. and is it needed? or can be completelly removed?

Reply:
Yes, we can remove it, default is empty